specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Encoding.json

```diff
@@ -1670,18 +1670,11 @@
             "description": "Restricts the set of items returned."
           },
           {
-            "name": "$top",
+            "name": "$orderby",
             "in": "query",
             "required": false,
-            "type": "integer",
-            "description": "Specifies a non-negative integer n that limits the number of items returned from a collection. The service returns the number of available items up to but not greater than the specified value n."
-          },
-          {
-            "name": "$skip",
-            "in": "query",
-            "required": false,
-            "type": "integer",
-            "description": "Specifies a non-negative integer n that excludes the first n items of the queried collection from the result. The service returns items starting at position n+1."
+            "type": "string",
+            "description": "Specifies the the key by which the result collection should be ordered."
```

Comment:
We are removing skip and top parameters here, are we doing this because this api-version never used to support them and we are just correcting it OR are we dropping this support? If second is the case then this is a breaking change that should be considered in the next api-version.

Reply:
It is the former.  This API has never supported skip and top.  They were added accidentally in the swagger and as part of implementing filter/orderby support they noticed and removed them.